cmd/apichecker/apicheck_test.go

```diff
@@ -0,0 +1,498 @@
+package main
+
+import (
+	"log"
+	"os"
+	"testing"
+	"reflect"
+	
+	exchange "github.com/thrasher-corp/gocryptotrader/exchanges"
+)
+
+func TestMain(m *testing.M) {
```

Comment:
It would be nice to be able to set your api key vars here, similar to how we do it with exchanges, then override the configData values

Reply:
havent addressed this one, not sure what to do bout it, coz the whole code is based on the fact that if someones doing testing it shouldnt update trello, but if api keys are set then it automatically updates trello so lemme know if u still want it changed

Reply:
Please add support for it, similar to how we do exchange testing. Default has apiKeys = "", but if a user enters their trello key/token, the tests should work to ensure that updating trello items work as expected